hotel_reservation/views/hotel_reservation_view.xml

```diff
@@ -0,0 +1,399 @@
+                        states="draft,confirm"
+                        class="oe_highlight"
+                        type="object"
+                    />
+                    <button
+                        name="set_to_draft_reservation"
+                        string="Set to Draft"
+                        states="cancel"
+                        class="oe_highlight"
+                        type="object"
+                    />
+                    <field name="state" widget="statusbar" />
+                </header>
+                <sheet>
+                    <div class="oe_button_box">
+                        <button
+                            name="open_folio_view"
+                            type="object"
+                            icon="fa-file-text-o"
+                            class="oe_stat_button"
+                            attrs="{'invisible': [('no_of_folio', '=', 0)]}"
+                        >
+                            <field
+                                name="no_of_folio"
+                                widget="statinfo"
+                                string="Folio"
+                            />
+                        </button>
+                    </div>
+                    <label for='reservation_no' string="Reservation No" />
+                    <h1>
+                        <field name="reservation_no" class="oe_inline" />
+                    </h1>
+                    <group>
+                        <group colspan="4" col="4">
+                            <field name="date_order" />
+                            <field name="warehouse_id" />
+                        </group>
+                        <separator />
+                        <group colspan="4" col="4">
+                            <field
+                                name="partner_id"
+                                domain="[('customer_rank','>', 0)]"
+                            />
+                            <field name="pricelist_id" col="2" />
+                            <field
+                                name="partner_order_id"
+                                domain="[('parent_id','=',partner_id)]"
+                            />
+                            <newline />
+                            <field
+                                name="partner_invoice_id"
+                                domain="[('parent_id','=',partner_id)]"
+                            />
+                            <field
+                                name="partner_shipping_id"
+                                domain="[('parent_id','=',partner_id)]"
+                                invisible="True"
+                            />
+                        </group>
+                        <group colspan="4" col="4">
+                            <field name="checkin" />
+                            <field name="checkout" />
+                            <field name="adults" required="1" />
```

Comment:
adults is an integer and I think we have validation so required is not needed

Reply:
Done.